src/Presentation/Salutations.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace WMDE\Fundraising\Frontend\Presentation;
+
+use Symfony\Component\Validator\Constraints as Assert;
+use Symfony\Component\Validator\Exception\ValidatorException;
+use Symfony\Component\Validator\Validation;
+
+class Salutations {
+
+	/**
+	 * @var array
+	 */
+	private array $salutations;
+
+	public function __construct( array $salutations ) {
+		if ( !$this->isValid( $salutations ) ) {
+			throw new ValidatorException( "Salutations are formatted incorrectly" );
+		}
+
+		$this->salutations = $salutations;
+	}
+
+	public function getList(): array {
+		return $this->salutations;
+	}
+
+	public function getSalutation( string $value ): ?array {
+		$data = array_filter( $this->salutations, fn( $salutation ) => $salutation['value'] == $value );
```

Comment:
Will we use `getSalutation` in our PHP code? If we foresee using it, we could optimize this lookup by using a key-value object in for salutations, where we drop `value` and it as key for the rest of the data instead.

Reply:
Yeah, we need it for the email greetings